src/three-components/shader-chunk/lights_physical_fragment.glsl.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * @license MIT
+ * @see https://github.com/mrdoob/three.js/blob/dev/LICENSE
+ */
+
+export const lightsChunk = /* glsl */ `
```

Comment:
Did we have to make modifications to this implementation?

Reply:
Yes, it's where I added the geometry normal anti-aliasing; all the stuff about specularRoughness.

Reply:
Added docs.